inst/shiny/server.R

```diff
@@ -91,7 +111,8 @@ load_data <- function() {
 server <- shinyServer(function(input, output, session) {
   #For theming
   #bs_themer()
-
+  observe_helpers()
```

Comment:
What is the purpose of calling `observe_helpers()` here? It's part of the shinyhelpers package not defined above. I commented out in my local repo for now without missing any functionality. If it's not a relic but needed, we should define the dependency properly.